### ISSUE TITLE
table-cell-padding with empty cells

### ISSUE THREAD

**Issue by BarryThePenguin** [wooorm/remark-lint#108]
With `"table-cell-padding": "padded"`

and the bellow markdown

```
| Content     | No Content |
| ----------- | ---------- |
| has content | `string`   |
| has content |            |
```

I receive the following warning

`  3:27  warning  Cell should be padded, isn’t  table-cell-padding  remark-lint`

Looks like it happens when there is an empty cell

**Comment by wooorm** [comment]
Hmm, whoops. Are you on the latest remark/remark-lint? Maybe this was [fixed in 5.2.0](https://github.com/wooorm/remark-lint/commit/2dc4d61f05ab2c6e7ff083785fa21aa9daf46ea8)

**Comment by BarryThePenguin** [comment]
It doesn't look like it :(

running

```js
var report = require('vfile-reporter');
var remark = require('remark');
var lint = require('remark-lint');

var file = remark().use(lint, {
  'table-cell-padding': 'padded'
}).process(`
# table-cell-padding test

| Content     | No Content |
| ----------- | ---------- |
| has content | 'string'   |
| has content |            |
`);

console.log(report(file));
```

with the latest dependancies still produces the same result

**Comment by kgryte** [comment]
@wooorm I am encountering the same bug. If a column heading is present, empty cells trigger a warning/error. If a column heading is empty, the warnings/errors go away.

**Comment by wooorm** [comment]
Thanks for waiting, both. Sorry, I’m swamped with work.

Turned out the rule was pretty broken... I rewrote it to actually work as expected. Could one of you verify it works now and does not cause unexpected changes?

**Comment by kgryte** [comment]
@wooorm Did you mean to publish a new version? Otherwise, to test this, we need to fork, create a `remark-lint` repo, and then `npm install` using the Git URL.

**Comment by kgryte** [comment]
Also, it appears that `remark-lint` version `5.3.0` was never published to `npm`.

**Comment by wooorm** [comment]
Lol, sorry about that!

About testing, no, I'd like for someone else to take a look before I push a version.
And, you can test locally by cloning the repo, npm installing, and than linking the subpackage to your project!

**Comment by BarryThePenguin** [comment]
Fixed here 😁  👍

**Comment by kgryte** [comment]
Thanks for testing @BarryThePenguin!

**Comment by kgryte** [comment]
@wooorm You okay publishing a new version or do you want to perform additional testing?

**Comment by wooorm** [comment]
Oh right, sorry about that 😬

**Comment by wooorm** [comment]
Released!

**Comment by kgryte** [comment]
Thanks, @wooorm!
